jenkins-scripts/sdformat-default-devel-homebrew-amd64.bash

```diff
@@ -4,7 +4,7 @@
 [[ -L ${0} ]] && SCRIPT_DIR=$(readlink ${0}) || SCRIPT_DIR=${0}
 SCRIPT_DIR="${SCRIPT_DIR%/*}"
 
-PIP_PACKAGES_NEEDED="psutil"
+PIP_PACKAGES_NEEDED="psutil pytest"
```

Comment:
Now that I think about it, `gz-math` is going to need this as well. Should I put this in `project-default-devel-homebrew-amd64.bash` instead?

Reply:
that sounds good to me

Reply:
Done in https://github.com/gazebo-tooling/release-tools/pull/759/commits/604cf628564277cdc5e7bb263b40b0f34048127d